package.json

```diff
@@ -21,6 +21,11 @@
   "type": "module",
   "devDependencies": {
     "@solidjs/testing-library": "^0.8.8",
+    "@vite-pwa/assets-generator": "^0.2.4",
+    "vite-plugin-pwa": "^0.20.0",
+    "sharp": "0.32.6",
+    "sharp-ico": "0.1.5",
+    "workbox-window": "^7.1.0",
```

Comment:
what do these 5 new dependencies do?

Reply:
These are required for the minimal config vite-pwa plugin. 

They are used generate the assets like splash screens, icons, etc. at build time and are not included in the applications bundle. 

There's an alternative way to use "resolutions" in package.json for the sharp and sharp-ico libs but I couldn't get them to work in this repo.

Reply:
It's a declarative plugin instead of an imperative config.